### PR TITLE
Fix ResponsiveContainer

### PR DESCRIPTION
**Pull request by HHongSeungWoo**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a storybook story or extended an existing story to show my changes
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by ckifer — CHANGES_REQUESTED** [review]
Curious what happens when multiple charts are added in one container. Even if that isn't supported, it still shouldn't break with a variable access issue like this might

File: src/component/ResponsiveContainer.tsx [hunk 1]
```diff
@@ -162,11 +163,22 @@ export const ResponsiveContainer = forwardRef<HTMLDivElement, Props>(
         aspect,
       );
 
+      const isCharts = getDisplayName(children.type).endsWith('Chart');
```
[the remark this answers is not in the document]
Comment (HHongSeungWoo) [reply src/component/ResponsiveContainer.tsx:166]: Even without this change, `ResponsiveContainer` has only allowed a single child.
I also don't want to use `getDisplayName`, but I don't have any other ideas at the moment

**Comment by HHongSeungWoo** [comment]
I have made changes to accept any type of children

**Review by ckifer — APPROVED** [review]
Unfortunately we kind of end up making this component more complex, but I think the benefits are worth it for sure - lets see what this looks like in chromatic

edit: looked good to me

**Comment by ckifer** [comment]
@HHongSeungWoo do you remember what this fixed?

**Pull request merged**